doc/README.md

```diff
@@ -50,9 +50,9 @@ Just provide the path to manifest as parameter:
 
 To submit a package to this repository, you should follow these steps:
 
-1. Follow the [Contributing](#contributing) guidelines below.
+1. Follow the [Contributing](/CONTRIBUTING.md) guidelines.
```

Comment:
```suggestion
1. Follow the [Contributing](../CONTRIBUTING.md) guidelines.
```

The URL seems "absolute" here. Can't sure if it can work properly. 😕

I would prefer using explicit relative links here.